src/functions/operatorname.js

```diff
@@ -69,7 +69,7 @@ defineFunction({
         identifier.setAttribute("mathvariant", "normal");
 
         const operator = new mathMLTree.MathNode("mo",
-            [mml.makeText("&ApplyFunction;", "text")]);
+            [mml.makeText("\u2061", "text")]);
```

Comment:
Maybe worth a comment that this is `<mo>&ApplyFunction;</mo>` as above?

Reply:
Will do.